buildfs.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+cpplint --recursive curvefs
+if [ $? -ne 0 ]
+then
+    echo "cpplint failed"
+    exit
+fi
+
+bazel build curvefs/proto:metaserver_cc_proto
+if [ $? -ne 0 ]
+then
+    echo "build metaserver proto failed"
+    exit
+fi
+bazel build curvefs/proto:mds_cc_proto
+if [ $? -ne 0 ]
+then
+    echo "build mds proto failed"
+    exit
+fi
+
+bazel build curvefs/src/metaserver:main  --copt -DHAVE_ZLIB=1 --compilation_mode=dbg -s --define=with_glog=true --define=libunwind=true --copt -DGFLAGS_NS=google --copt -Wno-error=format-security --copt -DUSE_BTHREAD_MUTEX --copt -DCURVEVERSION=${curve_version} --linkopt -L/usr/local/lib
```

Comment:
--compilation_mode=dbg，
does this just use for dev stage? release package will be debug mode or not?

Reply:
done